content/blog/2018/2018-08-31-useful-advice.md

```diff
@@ -21,10 +21,7 @@ Link: <https://pastebin.com/raw/R3fY11zY>
 To use, just copy contents of that snippet into javascript console when CCCC
```

Comment:
Also `<!--more-->` somewhere?..

Reply:
I didn't like the summary starting with "Update" so I added the first sentence of the original post as an explicit summary.

Reply:
I actually like summaries starting with the word `Update` (there are many examples in posts that I already cleaned up), as that shows the most important information makes people see that there is an update for the posts they already read.